### ISSUE TITLE
Adding VrcFury to an Avatar can Break Default VRChat Animations

### ISSUE THREAD

**Issue by chimerabyte** [VRCFury/VRCFury#164]
For some reason adding VrcFury to my avatar has been breaking my avatar. It only started today, but trying to go back a few versions doesn't fix the issue either. The animations only fix when I remove the VRCFury script from my avatar. It's happening even though the only thing I have the script doing is fixing anchor overrides and it was still happening when I removed the anchor overrides fix and left vrcfury there with nothing on it.

I'm attaching screenshots of what the issue looks like in game. First image is supposed to be a falling animation. Second is when walking around. The last might be related, but I'm unsure. I just noticed VRCFury was throwing this into the Unity logs when entering play mode. I've also left a copy/paste of the log.

![VRChat_2023-11-12_15-44-49 319_1920x1080](https://github.com/VRCFury/VRCFury/assets/59032948/7f76a93c-a14f-4bf4-bc30-33a7fbd56cab)
Falling animation is just the default Unity empty animation pose.

![VRChat_2023-11-12_15-45-11 343_1920x1080](https://github.com/VRCFury/VRCFury/assets/59032948/7da906d6-4383-468f-8152-7c7167682888)
The legs move in unison, stiff and floating in the air, rather than planting the feet on the ground.

![Screenshot 2023-11-12 154841](https://github.com/VRCFury/VRCFury/assets/59032948/ab03314c-5505-408d-8194-7e0166e887d3)

```
The referenced script on this Behaviour (Game Object '<null>') is missing!
UnityEngine.StackTraceUtility:ExtractStackTrace ()
VF.PreSaveVerifier:OnWillSaveAssets (string[]) (at Packages/com.vrcfury.vrcfury/Editor/VF/PreSaveVerifier.cs:29)
UnityEditor.AssetModificationProcessorInternal:OnWillSaveAssets (string[],string[]&,string[]&,bool)
VF.Builder.VRCFuryBuilder:SafeRun (VF.Builder.VFGameObject,VF.Builder.VFGameObject) (at Packages/com.vrcfury.vrcfury/Editor/VF/Builder/VRCFuryBuilder.cs:33)
VF.PlayModeTrigger:Rescan (UnityEngine.SceneManagement.Scene) (at Packages/com.vrcfury.vrcfury/Editor/VF/PlayModeTrigger.cs:80)
VF.PlayModeTrigger:OnSceneLoaded (UnityEngine.SceneManagement.Scene,UnityEngine.SceneManagement.LoadSceneMode) (at Packages/com.vrcfury.vrcfury/Editor/VF/PlayModeTrigger.cs:42)
UnityEngine.SceneManagement.SceneManager:Internal_SceneLoaded (UnityEngine.SceneManagement.Scene,UnityEngine.SceneManagement.LoadSceneMode)
```

**Comment by chimerabyte** [comment]
Adding more context to this. I first made this avatar on Nov. 3 and it worked fine then. So I think something between then and today broke things.

**Comment by SenkyDragon** [comment]
Can you provide a project that reproduces this issue? Join us on discord at https://vrcfury.com/discord

**Comment by chimerabyte** [comment]
I could provide my current project as-is, though it contains several paid assets. I'll join the discord as well so I can share screenshots or even screen-share the issue if I need to and see if it's just my project or something else. In the meantime I'll setup a new project with some free assets to see if it produces the same issue.

**Comment by SenkyDragon** [comment]
Fixed in 64429634
